boldgrid-backup.php

```diff
@@ -16,7 +16,7 @@
  *          Plugin Name: Total Upkeep
  *          Plugin URI: https://www.boldgrid.com/boldgrid-backup/
  *          Description: Automated backups, remote backup to Amazon S3 and Google Drive, stop website crashes before they happen and more. Total Upkeep is the backup solution you need.
- *          Version: 1.14.0-rc.3
+ *          Version: 1.14.0-rc.5
```

Comment:
Will the next one be rc6?

Reply:
Yeah. I just changed that so it was matching the current RC